### PR TITLE
Replace appcelerator relations

### PR DESCRIPTION
**Pull request by caspahouzer**
we have to clear this repo from Axway, Appcelerator Inc. and dead links

### CODE REVIEW

File: TitaniumKit.podspec [hunk 1]
```diff
@@ -4,20 +4,20 @@ Pod::Spec.new do |s|
 
   s.name         = "TitaniumKit"
   s.version      = "1.0.0"
-  s.summary      = "Axway Titanium iOS Core framework"
+  s.summary      = "TiDev Titanium iOS Core framework"
```
Comment (m1ga) [inline TitaniumKit.podspec:7]: Is this correct? Or just Titanium iOS...

File: TitaniumKit.podspec [hunk 2]
```diff
@@ -4,20 +4,20 @@ Pod::Spec.new do |s|
...
 
   s.description  = <<-DESC
-                   The Axway Titanium iOS Core framework.
+                   The TiDev Titanium iOS Core framework.
```
Comment (m1ga) [inline TitaniumKit.podspec:10]: Is this correct? Or just Titanium iOS...

File: apidoc/Titanium/CHANGELOG/1.6.0.mdoc [hunk 3]
```diff
@@ -9,25 +9,10 @@ We [addressed](https://appcelerator.lighthouseapp.com/projects/32238-titanium-mo
...
-We added 26 [features](https://appcelerator.lighthouseapp.com/projects/32238-titanium-mobile/tickets?q=milestone%3A%22Release+1.6.0%22+state%3Aresolved+tagged%3Aandroid+tagged%3Afeature&filter=). 
-
-Read the answer to this [Q&A](https://developer.appcelerator.com/question/112581/tableview-with-images-really-slow) post on our HTTP Response cache and other performance hints.
+We added 26 features. 
 
 Support for Android API 4 (1.6) is DEPRECATED as of 1.6.0. It will be removed in 1.7.0
 
-*Features*
-
-- Dimensions are supported in views nows. Android supports `mm`, `cm`, `in`, `px`, `pt`, `dp`, `dip`, `sp`, `sip`, and `%`. [Ticket 1125](https://appcelerator.lighthouseapp.com/projects/32238/tickets/1125-android-support-dimensions-in-layout)
```
Comment (m1ga) [inline apidoc/Titanium/CHANGELOG/1.6.0.mdoc:20]: keep the decription

File: apidoc/Titanium/CHANGELOG/1.6.0.mdoc [hunk 4]
```diff
@@ -37,18 +22,12 @@ Use a listener on the Window's "open" event to ensure that code gets executed af
 
 iOS Changes
 -----------
-We added 9 [features](https://appcelerator.lighthouseapp.com/projects/32238-titanium-mobile/tickets?q=milestone%3A%22Release+1.6.0%22+state%3Aresolved+tagged%3Aios+tagged%3Afeature&filter=), including:
```
Comment (m1ga) [inline apidoc/Titanium/CHANGELOG/1.6.0.mdoc:40]: keep the decription

File: iphone/TitaniumKit/docs.sh [hunk 5]
```diff
@@ -23,8 +23,8 @@ jazzy \
   --clean \
   --author Appcelerator \
```
Comment (m1ga) [inline iphone/TitaniumKit/docs.sh:24]: Tidev?

File: iphone/TitaniumKit/docs.sh [hunk 6]
```diff
@@ -23,8 +23,8 @@ jazzy \
...
   --author_url https://appcelerator.com \
```
Comment (m1ga) [inline iphone/TitaniumKit/docs.sh:25]: change the url?

File: tests/README.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 # Titanium Mocha Suite
 
-The unit testing app for [Axway Titanium](https://github.com/appcelerator/titanium_mobile), built using [TiMocha](https://github.com/tonylukasavage/ti-mocha).
+The unit testing app for [Axway Titanium](https://github.com/tidev/titanium_mobile), built using [TiMocha](https://github.com/tonylukasavage/ti-mocha).
```
Comment (m1ga) [inline tests/README.md:3]: remove [Axway] from the link

**Comment by caspahouzer** [comment]
Can this be merged?

**Comment by cb1kenobi** [comment]
@caspahouzer nope. Nobody has approved it yet. I can review it, but then something else has to give.

**Comment by caspahouzer** [comment]
I don't understand that process 🙈
@m1ga has reviewed it and had commented what I then had fixed. 

Because of that I have asked

**Comment by hansemannn** [comment]
@m1ga Please use the "Approve" button under "Files changed" > "Review changes". Every PR should have an official review approval, so it's easier to see who managed the review process.

**Comment by m1ga** [comment]
@hansemannn I know, but I wasn't doing a full review with approval :smile: I was just quickly checking the files for any mistakes. I Since there are changes in some source files you should run the test suite with this SDK to make sure nothing breaks and I didn't do that so I didn't want to approve it.

**Comment by caspahouzer** [comment]
> @hansemannn I know, but I wasn't doing a full review with approval 😄 I was just quickly checking the files for any mistakes. I Since there are changes in some source files you should run the test suite with this SDK to make sure nothing breaks and I didn't do that so I didn't want to approve it.

Changes are only made in comments. Dare at last 😄

File: README.md [hunk 8]
```diff
@@ -182,23 +182,23 @@ Community support and discussion about Titanium is available on Slack at [TiSlac
...
 
-[You can digitally sign the CLA](http://cla.appcelerator.com) online. Please indicate your email address in your first pull request so that we can make sure that will locate your CLA.  Once you've submitted it, you no longer need to send one for subsequent submissions.
+[You can digitally sign the CLA](https://github.com/tidev/organization-docs/blob/main/AUTHORIZED_CONTRIBUTORS.mdhttp://cla.appcelerator.com) online. Please indicate your email address in your first pull request so that we can make sure that will locate your CLA.  Once you've submitted it, you no longer need to send one for subsequent submissions.
```
Comment (m1ga) [inline README.md:201]: broken link

File: apidoc/Titanium/CHANGELOG/2.1.0.GA.mdoc [hunk 9]
```diff
@@ -289,7 +278,4 @@ to the unified documentation site for the Titanium Mobile SDK and Titanium Studi
     fix this problem.
 *   Display anomalies when a large number of tabs are open.
     
-For a complete list of documentation fixes in this release, see:
-
-[Fixed Documentation Issues in JIRA](https://jira.appcelerator.org/secure/IssueNavigator.jspa?mode=hide&requestId=13034)
-
+For a complete list of documentation fixes in this release, see:
```
Comment (m1ga) [inline apidoc/Titanium/CHANGELOG/2.1.0.GA.mdoc:281]: remove this line if the link below is gone

**Review by m1ga — COMMENTED** [review]
left some last comments. Test run fine, apps still build. So after that I can approve it from my side

**Review by m1ga — APPROVED** [review]
LGTM

**Pull request merged**